### ISSUE TITLE
failure to spawn node

### ISSUE THREAD

**Issue by ddrowley** [xqms/rosmon#102]
Ran into an issue where rosmon was failing to spawn a node. The log file reported:

```
Could not execute <blah blah blah> : Bad address
<blah> died from signal 6
```

Per execvp documentation, the list of args must be terminated by a NULL pointer, so adding this before the call to execvp (line 165 in shim.cpp) seems to resolve the issue:

```args.push_back((char*)NULL);```

**Comment by xqms** [comment]
Uh, that's embarrassing. I'm surprised that one slipped by for so long ;-)
Thanks for the report!

I'm tracking a fix in #103 and will try to release that as soon as possible.

**Comment by xqms** [comment]
Released as 2.2.1.
